packages/prop-house-webapp/src/components/RoundModules/index.tsx

```diff
@@ -113,7 +121,10 @@ const RoundModules: React.FC<{
               <Button
                 text={t('createYourProposal')}
                 bgColor={ButtonColor.Green}
-                onClick={() => navigate('/create', { state: { auction, community } })}
+                onClick={() => {
+                  dispatch(clearProposal());
```

Comment:
all these updates are just Prettier so ignore them.

This is the only line that matters.